packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/java/challengeintegrationsteps.md

```diff
@@ -0,0 +1,51 @@
+### 1 - 4: Sign-in and Select Authenticator
+
+The challenge flow follows the same first four steps as the [enrollment flow](#integrate-sdk-for-authenticator-enrollment):
+
+* Build a sign-in page on the client
+* Authenticate the user credentials
+* Handle the response from the sign-in flow
+* Display a list of possible authenticator factors
+
+### 5: Check Authenticator Status
+
+When the user selects the Google Authenticator factor, call the `selectAuthenticator()` method, to tell Okta which factor is being used.
+
+When in Challenge Flow, a call is made to `idxClient.SelectChallengeAuthenticatorAsync`, using its `selectAuthenticatorOptions` parameter to pass in the Google Authenticator factor ID.
```

Comment:
What is "Google Authenticator factor ID"? Is it the TOTP entered by the user? Or is it informing Okta of the selected authenticator, in which case, what's the differentiation with the sentence above (it can't be simply challenge flow because this is the challenge flow.) It also doesn't seem necesary to call out "when in challenge flow" because this is whole document is about the challenge flow.